app/assets/stylesheets/application/publishers.sass

```diff
@@ -72,12 +72,8 @@ body[data-controller="u2f_registrations"]
     .content-panel
       padding: 0 30px
 
-  .login-panel
```

Comment:
`.login-panel` is no longer used. It was simply doing some special-case padding and responsiveness stuff which has been accomplished through use of the grid.